spec/system/page_objects/components/topic_list.rb

```diff
@@ -46,6 +46,14 @@ def has_no_unread_badge?(topic)
         page.has_no_css?("#{topic_list_item_unread_badge(topic)}")
       end
 
+      def has_checkbox_selected_on_first_topic?
+        page.has_css?("#{TOPIC_LIST_ITEM_SELECTOR}:first-child input.bulk-select:checked")
```

Comment:
I think it would be helpful we renamed this method and used nth-child(1) here, so that you can pass in what topic number it is. This way future topic list tests could take advantage of it.

Reply:
Sure, good idea!